docs/admin/ldap/configure-krb.md

```diff
@@ -51,6 +51,12 @@ Finally, click on edit configuration and fill in the attributes.
 
 All defaults are taken from [OpenLDAP](https://www.openldap.org/).
 
+!!! note
+    Group mapping can be disabled by setting ```ldap_group_mapping_enabled=false``` in the [Configuration](../variables.md) UI.
+    When group mapping is disabled an administrator needs to activate each user from the [User Management](../user.md) page.
+
+    If group mapping is disabled then Account status in the configuration above should be set to Verified.
```

Comment:
`in the configuration above`
Maybe be more specific, it's the user management configuration?